src/runtime/pkg/magent/metrics.go

```diff
@@ -0,0 +1,297 @@
+package magent
+
+import (
+	"bytes"
+	"compress/gzip"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	mutils "github.com/kata-containers/kata-containers/src/runtime/pkg/utils"
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/expfmt"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+const (
+	kataRuntimeName              = "io.containerd.kata.v2"
```

Comment:
This is hard-coded in `cli/containerd-shim-kata-v2/main.go` so maybe you could find a way to define this variable in another packages that both this file and `cli/containerd-shim-kata-v2/main.go` can import?